### DISCUSSION TITLE
rustdesk.con file

### DISCUSSION THREAD

**Discussion by erict35000** [rustdesk/rustdesk-server-pro#181] [General]
Hi,

In the install script I see the nains rustdesk.conf file.

Where I can find it ?

Regards

**Comment by rustdesk** [comment]
It is nginx conf file.

https://github.com/rustdesk/rustdesk-server-pro/blob/c7ac7a628c03005fe0e645af00985aa9b6db6fcf/install.sh#L415

**Comment by erict35000** [comment]
Many thanks.

I didn't read the install script carefully :-)
